test/bridge.test.js

```diff
@@ -0,0 +1,64 @@
+const { describe, it } = require('mocha');
+const util = require('util');
+const assert = require('assert');
+const fs = require('fs-extra');
+const path = require('path');
+const exec = util.promisify(require('child_process').exec);
+const solc = require('solc');
+
+const getPath = (file) => path.resolve(__dirname, '..', 'contracts', file);
+
+describe('Bridge generator', () => {
+  it('generates a bridge file', async () => {
+    try {
+      // delete previously created interface
+      await fs.remove(getPath('Bridge.sol'));
+      // execute Terminal command
+      const { stdout } = await exec('node index.js');
+      // compare standart output with a certain string
+      assert.strictEqual(
+        stdout,
+        'The bridge was generated. Find it in the file ./contracts/Bridge.sol\n',
+      );
```

Comment:
should test more than just this - comparison of the actual output to expected values would be ideal.
... but for another patch!